### PR TITLE
Upgrade ember cli

### PR DESCRIPTION
**Pull request by SergeAstapov**

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
```
Comment (systembugtj) [inline .hound.yml]: what's the purpose of this file?

File: package.json [hunk 2]
```diff
@@ -1,54 +1,55 @@
 {
   "name": "ember-cli-mousewheel",
   "version": "0.1.5",
```
Comment (systembugtj) [inline package.json]: Please update the version.

File: .hound.yml (hunk 1 above)
Comment (SergeAstapov) [reply .hound.yml]: @systembugtj houndci-bot did not resolve esversion: 6 put into .jshintrc file in https://github.com/systembugtj/ember-cli-panzoom/pull/3
I might be overcomplicating - will remove it! Thanks!

File: package.json (hunk 2 above)
Comment (SergeAstapov) [reply package.json]: @systembugtj do you feel `0.1.6` would be fine? Since this is Ember CLI update does it make sense to bump up to 0.2.0?

Comment (systembugtj) [reply package.json]: since it's using different approach, 0.2.0 is better. 👍

Comment (systembugtj) [reply package.json]: Could you please update README also?

File: README.md [hunk 3]
```diff
@@ -4,23 +4,23 @@ This is an ember cli addon for jquery-mousewheel.
 
 ## Installation
 
-```bash
```
Comment (systembugtj) [inline README.md]: should keep this line.

**Comment by systembugtj** [comment]
@SergeAstapov Thanks for the upgrade. 👍

File: package.json (hunk 2 above)
Comment (systembugtj) [reply package.json]: I will bump the version.

**Comment by SergeAstapov** [comment]
@systembugtj Thank you for quick feedback!

**Pull request merged**